frontend/packages/process-editor/src/components/ConfigPanel/ConfigContent/EditActions/EditActions.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { useBpmnContext } from '../../../../contexts/BpmnContext';
+import { useTranslation } from 'react-i18next';
+import { StudioProperty } from '@studio/components';
+import type Modeling from 'bpmn-js/lib/features/modeling/Modeling';
+import type { ModdleElement } from 'bpmn-js/lib/BaseModeler';
+import type BpmnFactory from 'bpmn-js/lib/features/modeling/BpmnFactory';
+import { addNewActionToTask, getAvailablePredefinedActions } from './ActionsUtils';
+import { EditAction } from './EditAction';
+
+// Should we do anything with syncing the policy file or just advice to app developer to navigate to policy editor?
+// Maybe not in the first implementation round anyway --> new issue
```

Comment:
Are these comments supposed to be there?

Reply:
No, since we now have created the issue for adding a short cut to the policy editor we are good now! Will remove it 😎